packages/design-system/src/Popover/Popover.stories.tsx

```diff
@@ -0,0 +1,101 @@
+import React from "react";
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+
+import {
+  Popover,
+  PopoverBody,
+  PopoverContent,
+  PopoverHeader,
+  PopoverTrigger,
+} from "./Popover";
+import { Button } from "../Button";
+import { Input } from "../Input";
+
+export default {
+  title: "Design System/Popover",
+  component: Popover,
+  subcomponents: {
+    PopoverBody,
+    PopoverContent,
+    PopoverHeader,
+  },
+} as ComponentMeta<typeof Popover>;
+
+// eslint-disable-next-line react/function-component-definition
+const PopoverHeaderTemplate: ComponentStory<typeof PopoverHeader> = (args) => {
+  return (
+    <Popover>
+      <PopoverTrigger>
+        <Button>Open popover</Button>
+      </PopoverTrigger>
+      <PopoverContent size="sm">
+        <PopoverHeader {...args} />
+      </PopoverContent>
+    </Popover>
+  );
+};
+
+export const PopoverHeaderStory = PopoverHeaderTemplate.bind({});
+PopoverHeaderStory.storyName = "Header";
+PopoverHeaderStory.args = {
+  children: "JS Libraries",
+  isClosable: true,
+};
+
+const PopoverContentTemplate: ComponentStory<typeof PopoverHeader> = (args) => {
+  return (
+    <Popover>
+      <PopoverTrigger>
+        <Button>Open popover</Button>
+      </PopoverTrigger>
+      <PopoverContent {...args} />
+    </Popover>
+  );
+};
+
+export const PopoverContentStory = PopoverContentTemplate.bind({});
+PopoverContentStory.storyName = "Content";
+PopoverContentStory.args = {
+  size: "sm",
+  children: (
+    <>
```

Comment:
This should be a `Popover` right?

Reply:
Yep, will update the story.

Reply:
Ah, I misunderstood your comment. No, that line is fine being a fragment, because Popover has been defined in it's template.

Reply:
Got it